src/components/App.tsx

```diff
@@ -716,12 +729,18 @@ class App extends React.Component<ExcalidrawProps, AppState> {
       commitToHistory: true,
     });
 
-    const addToLibraryUrl = new URLSearchParams(window.location.search).get(
-      "addLibrary",
-    );
+    const libraryUrl =
+      // current
+      new URLSearchParams(window.location.hash.slice(1)).get(
+        URL_HASH_KEYS.addLibrary,
+      ) ||
+      // legacy, kept for compat reasons
+      new URLSearchParams(window.location.search).get(
```

Comment:
just to confirm this will make it a non breaking change for the package right ?

Reply:
Actually, no. The package will use the new hash workflow as well. But this code path was added for compatibility with older client versions out there (due to the ServiceWorker thing).

Should I add a `libraryScheme: "hash" | "query" = "query"` prop instead to make it non-breaking?

Reply:
That said, depending on https://github.com/excalidraw/excalidraw/issues/3321, it might not matter.

Reply:
Got it, I was worried about host would have to use the API `importLibrary` to make it work, but since thats not the case its just the we are now appending hash instead of query param so should be fine and host can optimize adding hashChange listener to avoid refresh.